portal/src/main/webapp/WEB-INF/jsp/global/global_variables.jsp

```diff
@@ -87,21 +91,58 @@
 
     //Info about queried cancer study
     ArrayList<CancerStudy> cancerStudies = (ArrayList<CancerStudy>)request.getAttribute(QueryBuilder.CANCER_TYPES_INTERNAL);
-    String cancerTypeId = (String) request.getAttribute(QueryBuilder.CANCER_STUDY_ID);
-    CancerStudy cancerStudy = cancerStudies.get(0);
+  //  String cancerTypeId = (String) request.getAttribute(QueryBuilder.CANCER_STUDY_ID);
+  //  CancerStudy cancerStudy = cancerStudies.get(0);
+    String[] cancerStudyIdList = (String[])request.getAttribute(QueryBuilder.CANCER_STUDY_LIST);
+	String cancerStudyIdListString = "";
+    HashMap<String, Boolean> studyMap = new HashMap<String, Boolean>();
+	for (String studyId : cancerStudyIdList) {
+		studyMap.put(studyId, Boolean.TRUE);
+	}
+	//CancerStudy cancerStudy = cancerStudies.get(0);
+    String mutationProfileIDs = "";
+    ArrayList<CancerStudy> selectedCancerStudies = new ArrayList<CancerStudy>();
+    boolean showIGVtab = false;
+    boolean has_survival = false;
```

Comment:
It is a Java coding convention to use camel case for variable naming. i.e: `hasSurvival` is prefered to `has_survival`